### PR TITLE
docs: update Twilio and WorkOS docs

### PR DESCRIPTION
**Pull request by J0**
## What kind of change does this PR introduce?

Updates Twilio and WorkOS docs

## What is the current behavior?

Twilio and WorkOS docs might be slightly stale or unclear

## What is the new behavior?

Hopefully clearer docs

## Additional context

NA

### CODE REVIEW

File: web/docs/guides/auth/auth-twilio.mdx [hunk 1]
```diff
@@ -9,12 +9,12 @@ import TabItem from '@theme/TabItem'
 
 ## Overview
 
-In this guide we'll show you how to authenticate your users with SMS based OTP (One-Time Password) tokens.
+In this guide we'll show you how to authenticate your users with SMS based One-Time Password(OTP) tokens.
```
Comment (liaujianjie) [inline web/docs/guides/auth/auth-twilio.mdx:12]: I think you forgot a space here 👀 

```suggestion
In this guide we'll show you how to authenticate your users with SMS based One-Time Password (OTP) tokens.
```

File: web/docs/guides/auth/auth-twilio.mdx [hunk 2]
```diff
@@ -89,11 +89,11 @@ The SMS message sent to a phone containing an OTP code can be customized. This i
...
+In this scenario we'll be using the user's mobile phone number and a corrsesponding password as an alternative to signing up with an email address. Note: Please deeply consider the potential security implications when signing up with a combination ofphone and passsword. It is not uncommon for mobile phone numbers to be recycled by phone networks when users cancel their phone contracts or move countries, therefore granting access to the user's account to the subsequent owner of the phone number. In the near future Supabase will support multifactor auth, which will mitigate this risk, but for now you may want to consider allowing your users to recover their account by some other means in an emergency.
```
Comment (J0) [inline web/docs/guides/auth/auth-twilio.mdx:96]:
```suggestion
In this scenario we'll be using the user's mobile phone number and a corresponding password as an alternative to signing up with an email address. Note: Please deeply consider the potential security implications when signing up with a combination ofphone and passsword. It is not uncommon for mobile phone numbers to be recycled by phone networks when users cancel their phone contracts or move countries, therefore granting access to the user's account to the subsequent owner of the phone number. In the near future Supabase will support multifactor auth, which will mitigate this risk, but for now you may want to consider allowing your users to recover their account by some other means in an emergency.
```

Comment (J0) [inline web/docs/guides/auth/auth-twilio.mdx:96]:
```suggestion
In this scenario we'll be using the user's mobile phone number and a corrsesponding password as an alternative to signing up with an email address. Note: Please deeply consider the potential security implications when signing up with a combination ofphone and passsword. It is not uncommon for mobile phone numbers to be recycled by phone networks when users cancel their phone contracts or move countries, therefore granting access to the user's account to the subsequent owner of the phone number. In the near future Supabase will support multifactor authentication, which will mitigate this risk, but for now you may want to consider allowing your users to recover their account by some other means in an emergency.
```

File: web/docs/guides/auth/auth-twilio.mdx (hunk 1 above)
Comment (J0) [reply web/docs/guides/auth/auth-twilio.mdx:12]: You're right. Thanks for the catch!

File: web/docs/guides/auth/auth-workos.mdx [hunk 3]
```diff
@@ -11,12 +11,16 @@ To enable WorkOS Auth for your project, you need to set up WorkOS OAuth applicat
...
 
-WorkOS OAuth consists of four broad steps:
+In this guide, we will cover how to use Supabase OAuth with WorkOS to implement Single-Sign-On(SSO).
+
+The procedure consists of five broad steps:
 
 - Create a new organization from your WorkOS Dashboard.
 - Obtain the `Client ID` from the Configuration tab and configure redirect URI.
 - Obtain the `WorkOS Secret` from the credentials tab.
+- Connect a WorkOS Connected Identity Provider
```
Comment (J0) [inline web/docs/guides/auth/auth-workos.mdx:21]:
```suggestion
- Connect a WorkOS Supported Identity Provider
```

File: web/docs/guides/auth/auth-twilio.mdx [hunk 4]
```diff
@@ -89,11 +89,11 @@ The SMS message sent to a phone containing an OTP code can be customized. This i
...
+In this scenario we'll be using the user's mobile phone number and a corresponding password as an alternative to signing up with an email address. Note: Please deeply consider the potential security implications when signing up with a combination ofphone and passsword. It is not uncommon for mobile phone numbers to be recycled by phone networks when users cancel their phone contracts or move countries, therefore granting access to the user's account to the subsequent owner of the phone number. In the near future Supabase will support multifactor auth, which will mitigate this risk, but for now you may want to consider allowing your users to recover their account by some other means in an emergency.
```
Comment (J0) [inline web/docs/guides/auth/auth-twilio.mdx:96]:
```suggestion
In this scenario we'll be using the user's mobile phone number and a corresponding password as an alternative to signing up with an email address. Note: please thoroughly consider potential security implications when signing up with a combination of phone number and password. Phone numbers are sometimes recycled by phone networks when users cancel their phone contracts or move countries, thereby granting access to the user's account to the subsequent owner of the phone number. In the near future Supabase will support multifactor authentication, which will mitigate this risk, but for now you may want to consider allowing your users to recover their account by some other means in an emergency.
```

**Review by kangmingtay — APPROVED** [review]

**Pull request merged**